anvil-python/anvil/commands/juju.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2024 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from os import environ
+import os.path
+import subprocess
+
+from rich.status import Status
+from sunbeam.jobs.common import BaseStep, Result, ResultType
+
+
+class JujuAddSshKeyStep(BaseStep):
```

Comment:
Nit:

```suggestion
class JujuAddSSHKeyStep(BaseStep):
```

If you agree with the change you also need to change it to `provider/local/commands.py`; import+usage